lms/templates/navigation-edx.html

```diff
@@ -142,7 +142,7 @@ <h1 class="logo" itemscope="" itemtype="http://schema.org/Organization">
 <![endif]-->
 % endif
 
-%if not user.is_authenticated():
+%if not enable_require_js and not user.is_authenticated():
   <%include file="forgot_password_modal.html" />
```

Comment:
Why is this not needed if ``enable_require_js`` is true? A comment would be helpful.